autosubmit_api/performance/performance_metrics.py

```diff
@@ -151,6 +184,22 @@ def _calculate_CHSY(self):
       CHSY = sum(job.CHSY for job in self.sim_jobs_valid)/len(self.sim_jobs_valid)
       return round(CHSY, 4)
     return 0
+  
+  def _calculate_sum_yps(self):
+    return sum(job.years_per_sim for job in self.sim_jobs_valid)
+  
+  def _calculate_sum_energy(self):
+    return sum(job.energy for job in self.sim_jobs_valid)
+  
+  def _calculate_sum_footprint(self):
+    if self.sim_jobs_platform == "":
+      return 0.0
+    return sum(job.energy * self.sim_platform_CF * self.sim_platform_PUE for job in self.sim_jobs_valid) / UNITS_CONVERSOR_ENERGY
```

Comment:
Maybe you can use a function you already made, like this:

```suggestion
    return self._calculate_sum_energy() * self.sim_platform_CF * self.sim_platform_PUE / UNITS_CONVERSOR_ENERGY
```

Now, we can argue about the order of the multiplications and divisions to prevent Python from doing slow Big Number multiplication or losing precision due to many decimals, but I think it can be done later.

Reply:
Sounds good! What if instead of using `self._calculate_sum_energy()` I directly use the variable with the total energy `self.valid_sim_energy_sum`?

Reply:
Sure!

Reply:
I will change the order of operations to be like `def _calculate_sim_job_footprint(self, simjob: SimJob):` to maintain consistency. Like you said, later it can be discussed the final order.